bin/fonto-benchmark-runner

```diff
@@ -1,7 +1,17 @@
-#!/usr/bin/env ts-node
+#!/usr/bin/env node
+
+console.log('!!!');
 
 'use strict';
 
+require('ts-node').register({
+	"extends": `${process.cwd()}/tsconfig.json`,
```

Comment:
Do we need to test whether this file exists?

Reply:
I think we do not need to check it. I tested it in a new project without tsconfig.json and it worked fine.